config/v1/0000_10_config-operator_01_infrastructure-Default.crd.yaml

```diff
@@ -561,6 +561,10 @@ spec:
                         region:
                           description: region holds the default Power VS region for new Power VS resources created by the cluster.
                           type: string
+                        resourceGroup:
+                          description: ResourceGroup is the resource group for new IBMCloud resources created for the cluster. It is created by user to organise the resources in IBMCloud by grouping them More about resource group https://cloud.ibm.com/docs/account?topic=account-rgs It will get read by cluster-image-registry-operator to set up COS Instance in IBMCloud
```

Comment:
cleaning up some grammar and punctuation:

`ResourceGroup is the resource group name for new IBMCloud resources created for a cluster.  The resource group specified here will be used by cluster-image-registry-operator to set up a COS Instance in IBMCloud for the cluster registry. More about resource groups can be found here: https://cloud.ibm.com/docs/account?topic=account-rgs. `

Reply:
Used this as comment for var.